cli/volume.go

```diff
@@ -304,10 +311,28 @@ func (v *VolumeListCommand) runVolumeList(args []string) error {
 	}
 
 	display := v.cli.NewTableDisplay()
-	display.AddRow([]string{"Name:"})
-
-	for _, v := range volumeList.Volumes {
-		display.AddRow([]string{v.Name})
+	displayHead := []string{"DRIVER", "VOLUME NAME"}
```

Comment:
Could you help to change `VOLUME NAME` into `VOLUME_NAME` to avoid others to think that `NAME` is another column? @rudyfly